src/parser/core/modules/ResourceGraphs/ResourceGraphs.tsx

```diff
@@ -49,6 +51,11 @@ const DEFAULT_ROW_HEIGHT: number = 64
 export interface ResourceGroupOptions extends ResourceGraphOptions {
 	/** The handle for this group of data */
 	handle: string,
+	/** Should the group display it's gauges stacked on top of each other, instead of all starting from the 0 baseline (ie. RPR's Death Gauge to help it look like it does in-game)
+	 * 		NOTE: If using this option, all gauges within the resource group should have a constant and identical maximum or it's gonna look hella weird.
+	 * 			Also recommended to use the forceCollapsed option along with this (unless debugging) since that'll also look weird.
+	  */
```

Comment:
hmmmmm

this is effectively simulating a stacked graph, but with all of the timeline infrastructure getting in the way while you're doing it.

the library being used for graph rendering, visx, has [a utility for generating data pre-stacked](https://airbnb.io/visx/stacked-areas) - it'll probably take a bit of massaging to make it work (primarily around the clip path handling), but it'd be good to lean on.

bonus, if you put the stacked graph on the group timeline row, then in debug you can render the individual gauges to child rows and it won't have wonky baselines

Reply:
I spent a... frustratingly long time trying to make the StackedArea work before falling back to this as a "dumb but it works" option. Willing to keep tinkering with things to see if I can figure out what it was I was too dumb to figure out at first tho.